Dockerfile

```diff
@@ -1,11 +1,17 @@
 # syntax=docker/dockerfile:experimental
 
-FROM golang:1.13.4-buster
+FROM golang:1.13.4-buster AS builder
 
 RUN mkdir -p /go/src/github.com/ava-labs
-
-WORKDIR $GOPATH/src/github.com/ava-labs/
+WORKDIR /src/github.com/ava-labs/
 COPY . gecko
-
-WORKDIR $GOPATH/src/github.com/ava-labs/gecko
+WORKDIR /src/github.com/ava-labs/gecko
 RUN ./scripts/build.sh
+
+FROM nginx
+
+COPY --from=builder /src/github.com/ava-labs/gecko/build /build
```

Comment:
you don't need to copy it to the `/build` directory in the target container, consider to use `/gecko` or `/ava` instead.

Reply:
ok. thanks. we could investigate this later